Failures in the "Build Windows Native Image" CI step

The last two CI runs (4 hours and 11 hours ago) have failed for the "Build Windows Native Image" step failed with the same error message:
```
 $ node build-image.js
  google-closure-compiler-windows building image
Error: spawn EINVAL
    at ChildProcess.spawn (node:internal/child_process:421:11)
    at spawn (node:child_process:761:9)
    at D:\a\closure-compiler-npm\closure-compiler-npm\build-scripts\run-command.js:53:23
    at new Promise (<anonymous>)
    at runCommand (D:\a\closure-compiler-npm\closure-compiler-npm\build-scripts\run-command.js:49:19)
    at Object.<anonymous> (D:\a\closure-compiler-npm\closure-compiler-npm\build-scripts\graal.js:60:1)
    at Module._compile (node:internal/modules/cjs/loader:1369:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Module._load (node:internal/modules/cjs/loader:1022:12) {
  errno: -4071,
  code: 'EINVAL',
  syscall: 'spawn'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

It seems many of these CI steps are a bit flaky, so hopefully that's the case, but filing this issue to track just in case.

--- a/build-scripts/graal.js
+++ b/build-scripts/graal.js
@@ -57,7 +57,12 @@ const NATIVE_IMAGE_BUILD_ARGS = [
   path.resolve(process.cwd(), 'compiler.jar')
 ];
 
-runCommand(`native-image${process.platform === 'win32' ? '.cmd' : ''}`, NATIVE_IMAGE_BUILD_ARGS)
+const spawnOpts = {};
+if (process.platform === 'win32') {
+  spawnOpts.shell = true;
+}
+
+runCommand(`native-image${process.platform === 'win32' ? '.cmd' : ''}`, NATIVE_IMAGE_BUILD_ARGS, spawnOpts)
     .catch(e => {
       console.error(e);
       process.exit(1);

